packages/harpguru-core/src/components/hole-number-row/hole-number-row.test.tsx

```diff
@@ -8,8 +8,19 @@ import { HoleNumberRow } from './hole-number-row'
 
 jest.mock('reactn')
 const mockUseGlobal = useGlobal as jest.Mock
+// TODO: Should be shared resource
```

Comment:
Already addressed in a code review above, but this comment needs removing and the code modifying here too.

Reply:
Done